rrule/tests/rrule.rs

```diff
@@ -4226,113 +3323,8 @@ fn secondly_by_month_and_yearday() {
     );
 }
 
+#[cfg(feature = "no-validation-limits")]
```

Comment:
What about these `no-validation-limits` tests @fmeringdal ?

Reply:
I think they should stay. Any reasons we would want to ignore/remove them?

Reply:
No, because I think you removed them in your PR #52

Reply:
Maybe I am blind, but I can't see that I removed these tests. I can't think of any reasons to remove them, so let's keep them.